lib/quantum/normalizer.ex

```diff
@@ -28,6 +32,12 @@ defmodule Quantum.Normalizer do
   defp normalize_job({job_name, %Quantum.Job{} = job}) do
     # Sets defauts for job if necessary
     job_name |> job_opts([]) |> Map.merge(%{job | name: job_name})
+    |> Map.put(
+        :nodes,
+        case job.nodes do
+          nil -> default_nodes
```

Comment:
This way we set the default value only when there is no value provided, i.e., th evalue is nil, whilst Map.merge overrides the value, thus it is not a suitable mechanism for defaulting, honoring user defined value, if any.